molecule/default/verify.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Verify role
+  hosts: all
+  become: yes
+
+  tasks:
```

Comment:
I don't know that there's anything we can do to test this functionally. All the role does is upload files. For this one, if converge passes, i think we're fine.